jina/peapods/container.py

```diff
@@ -69,29 +72,46 @@ def post_init(self):
                                                       name=self.name,
                                                       volumes=_volumes,
                                                       network_mode=net_mode,
-                                                      entrypoint=self.args.entrypoint,
-                                                      # publish_all_ports=True # This looks like something I would
-                                                      # activate
-                                                      )
+                                                      entrypoint=self.args.entrypoint)
+
+        # Related to potential docker-in-docker communication. If `ContainerPea` lives already inside a container.
+        # it will need to communicate using the `bridge` network.
+        if platform in ('linux', 'linux2'):
+            bridge_network = self._client.networks.get('bridge')
+            if bridge_network:
+                self.ctrl_addr, self.ctrl_with_ipc = Zmqlet.get_ctrl_address(
```

Comment:
apparently using `bridge` gateway seems to allow the `ContainerPea` to always connect with the Pea in container.

Reply:
## Background

```
hubapp (container#1)
    |- gateway
    |- pod1
    |    |- pea1_1
    |        |- hubpod (container#2)
    |- ...
```
## Goal
What is the goal here? 
Do we want to pass a message from `gateway` to `container#2`?
Or do we want to send the ctrl signal directly to `container#2`?

Reply:
We want to be able to send control message from `pea_1` to `hubpod (container #2) (I think I understood the schema.